command-expansion-server/src/polyfills.ts

```diff
@@ -0,0 +1,5 @@
+import { Temporal as TemporalPolyfill, toTemporalInstant } from '@js-temporal/polyfill';
+
+globalThis.Temporal = TemporalPolyfill;
```

Comment:
I'm not sure I understand the need to set a global variable here. This is setting a global variable in the command-expansion-server, not in the user code.

Reply:
Temporal is the upcoming standard that isn't in ecmascript yet, so we have to poly fill it. This is placing it into the global namespace so that it exists globally as it will when it gets in.